csrc/executor.cpp

```diff
@@ -631,7 +631,7 @@ class ForwardTraverseFromAllocToRFactor {
     // view tensor
     int64_t dim = std::distance(frontier_.begin(), in_it);
     std::vector<int64_t> new_shape;
-    for (auto i : c10::irange(tensor_.dim())) {
+    for (auto i : std::views::iota((int64_t)0, tensor_.dim())) {
```

Comment:
0LL. In general, avoid C-style cast because it's too permissive.

Reply:
Unfortunately 0LL does not work, at least not for gcc 13.2.1. 0L should work, but IIUC, it will not work on MSVC. It is unfortunate that C++ uses vague names `int`, `long`, `long long` for integers and does not specify integer sizes.